native/packages/react-native-bpk-component-banner-alert/src/BpkBannerAlert.ios.js

```diff
@@ -47,7 +47,6 @@ import {
 
 const STYLES = StyleSheet.create({
   border: {
-    minHeight: spacingXl, // animate height is janky if we don't provide this ¯\_(ツ)_/¯
```

Comment:
This is no longer a problem - our new method of animating doesn't surface this bug.